packages/surveys/src/components/WelcomeQuestion.tsx

```diff
@@ -0,0 +1,53 @@
+import { TResponseData } from "@formbricks/types/v1/responses";
+import type { TSurveyWelcomeQuestion } from "@formbricks/types/v1/surveys";
+import Headline from "./Headline";
+import HtmlBody from "./HtmlBody";
+import SubmitButton from "./SubmitButton";
+
+interface WelcomeQuestionProps {
+  question: TSurveyWelcomeQuestion;
+  value: string | number | string[];
+  onChange: (responseData: TResponseData) => void;
+  onSubmit: (data: TResponseData) => void;
+  isLastQuestion: boolean;
+  brandColor: string;
+}
+
+export default function WelcomeQuestion({
+  question,
+  onSubmit,
+  isLastQuestion,
+  brandColor,
+}: WelcomeQuestionProps) {
+  return (
+    <div>
+      {question.companyLogo && (
+        <img
```

Comment:
Please use the Image component from `next/image` here 🙏

Reply:
next/Image not working for some reason